quality_of_service_demo/rclcpp/src/message_lost_talker.cpp

```diff
@@ -37,6 +37,7 @@ void print_usage()
     "Usage: message_lost_talker [-h] [-s SIZE]\n\n"
     "optional arguments:\n"
     "\t-h:                           Print this help message.\n"
+    "\t-4:                           Timer rate in Hz, default to 0.3 Hz\n"
```

Comment:
```suggestion
    "\t-r:                           Timer rate in Hz, default to 0.3 Hz\n"
```

Assuming this was supposed to be `-r` and not `-4`